app/models/Users.scala

```diff
@@ -60,6 +60,10 @@ object Users {
     findSingleUser { user => user.email === email }
   }
 
+  def findById(id: Int): Option[User] = {
+    findSingleUser { user => user.id === id }
+  }
```

Comment:
Hm. Given that it's such a common query to get something by the primary ID, I kind of assumed there'd be a built-in method for it.